scaleway/resource_rdb_read_replica.go

```diff
@@ -88,7 +88,8 @@ func resourceScalewayRdbReadReplica() *schema.Resource {
 						"service_ip": {
 							Type:         schema.TypeString,
 							Description:  "Endpoint IPv4 address with a CIDR notation. Check documentation about IP and subnet limitations. (IP network).",
-							Required:     true,
+							Optional:     true,
```

Comment:
Can you test this? `serivice_ip` seems required in `expandReadReplicaEndpointsSpecPrivateNetwork` or else it will panic

Reply:
The `service_ip` is optional on the last [sdk](https://github.com/scaleway/scaleway-sdk-go/blob/master/api/rdb/v1/rdb_sdk.go#L913) version